### DISCUSSION TITLE
Registration: How to do inter-subject registration where moving/target images have different affine

### DISCUSSION THREAD

**Discussion by dyollb** [Project-MONAI/MONAI#8234] [Q&A]
I am trying to adapt the monai registration tutorials to my data. Briefly, my data is a collection of head MRI of different subjects and one reference atlas.

- I am trying to use the [learn2reg_nlst_paired_lung_ct](https://github.com/Project-MONAI/tutorials/blob/main/3d_registration/learn2reg_nlst_paired_lung_ct.ipynb) tutorial to load my data
- Using ants/SimpleITK I have already registered the atlas to all these subjects. 
- I extracted points in the atlas and transformed them to the subject images (using SimpleITK)
- the original *learn2reg* data used in the tutorial seems to expect points in "image space" (basically image indices assuming e.g. RAS ordering). I extract my points in the same way

The first thing I observed is that my validation TRE improves, but the validation Dice does not. Trying to understand this, I realize the original tutorial is using images that all have been stripped of the original scanner coordinate system. The image affine matrices are identity. I applied the same "strip-scanner-coordinates" to my data, and now the validation dice improves during training. 

Looking at other MONAI tutorials, it seems the moving and fixed always have an identity matrix as affine, or they have the same affine matrix. Even the GlobalNet example is not learning the affine matrix to align images in physical space/scanner coordinates.

Maybe I am confused because the ITK framework (ants/SimpleITK are based on ITK) typically finds transforms in "physical space" and MONAI finds transforms (DDF & affine) in "continuous index space". 

Maybe it would be useful to have some tooling to convert "continuous index space" DDF or affine to "physical space"...?

**Comment by aylward** [comment]
You might find the https://github.com/Project-MONAI/MONAI/discussions/8236 discussion to be helpful. 

You bring up a great point about physical space versus image space.   ITK has some great discussions on why registration must be performed in physical space (not image space).   Hopefully we can get that in MONAI soon too.

**Reply by dyollb** [reply]
Thanks @aylward. Sounds promising.

**Comment by aylward** [comment]
Here is a CMU course on image registration : https://www.cs.cmu.edu/~galeotti/methods_course/ITK_Registration.pdf
